src/tools/kotlin-entity-generator.ts

```diff
@@ -323,8 +282,8 @@ var ${escaped}: ${type}
         }`;
   }
 
-  private mapFields(mapper: (arg: KotlinSchemaField) => string): string[] {
-    return this.entityDescriptor.fields.map(mapper);
+  private mapFields(mapper: (arg: ExtendedSchemaField) => string): string[] {
```

Comment:
You can use the definition from `hot.ts`: `mapper: Mapper<ExtendedSchemaField, string>` if you'd like.